### PR TITLE
LRQA-72177 | master

### PR DESCRIPTION
**Pull request by melodywu**
https://issues.liferay.com/browse/LRQA-72177

### CODE REVIEW

**Comment by melodywu** [comment]
ci:test:sf

**Comment by melodywu** [comment]
ci:test:sf

**Comment by melodywu** [comment]
ci:test:sf

**Review by patriciajeanperez — COMMENTED** [review]
@melodywu Looks good! We just need to remove redundant assertions. Thank you!

**Comment by patriciajeanperez** [comment]
ci:close

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/users/gdpr/GDPR.testcase [hunk 1]
```diff
@@ -1767,67 +1767,45 @@ definition {
...
-					for (var unlistedEntries : list "Announcements,Blogs,Bookmarks,Contacts Center,Documents and Media,Forms,Web Content,Message Boards") {
-						LexiconEntry.viewNoEntry(rowEntry = "${unlistedEntries}");
-					}
 				}
 				else {
-					Pagination.viewResults(results = "Showing 1 to 9 of 9 entries.");
+					Pagination.viewResults(results = "Showing 1 to 8 of 8 entries.");
 
-					for (var allEntries : list "Announcements,Blogs,Bookmarks,Contacts Center,Documents and Media,Forms,Web Content,Message Boards,Wiki") {
+					for (var allEntries : list "Announcements,Blogs,Contacts Center,Documents and Media,Forms,Web Content,Message Boards,Wiki") {
```
[the remark this answers is not in the document]
Comment (melodywu) [reply portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/users/gdpr/GDPR.testcase:1808]: they are checking 20 40 and 60. Don't we need them anymore?

**Pull request closed without merging**